index.html

```diff
@@ -30,11 +30,11 @@
                </div>
                <div class="social_icon_container">
                    <ul>
-                       <li><a href="#" class="twitter"><img src="images/twitter.png" alt="twiter icon"></a></li>
-                       <li><a href="#" class="linkedin"><img src="images/Linkedin icon.png" alt="Linkedin icon"></a></li>
-                       <li><a href="#" class="medium"><img src="images/medium.png" alt="medium icon"></a></li>
-                       <li><a href="#" class="github"><img src="images/github.png" alt="github icon"></a></li>
-                       <li><a href="#" class="angellist"><img src="images/angellist.png" alt="angellist icon"></a></li> 
+                       <li><a href="#" class="twitter"><img src="images/twitter.png" alt="Twiter login"></a></li>
+                       <li><a href="#" class="linkedin"><img src="images/Linkedin icon.png" alt="Linkedin login"></a></li>
+                       <li><a href="#" class="medium"><img src="images/medium.png" alt="medium login"></a></li>
+                       <li><a href="#" class="github"><img src="images/github.png" alt="github login"></a></li>
+                       <li><a href="#" class="angellist"><img src="images/angellist.png" alt="angellist login"></a></li> 
```

Comment:
- [ ] I noticed there is horizontal scrolling and texts are being cut off on the mobile version of your application. Kindly fix this accessibility issue. 

![chrome-capture-2022-11-14](https://user-images.githubusercontent.com/77106962/207469859-24540b0f-dbb3-4b2a-afa7-1bb36cf2cabc.gif)

Reply:
@Gambit142 Please be informed that we have fixed the horizontal scrolling issue. Thank you for your time and efforts.

Reply:
- [ ] I noticed that you tried to fix this horizontal scrolling issue by setting the `<body>` element width to a fixed `375px`. Please avoid fixed units as much as possible! This is because fixed units don't give you as much flexibility in styling a responsive design. 🎨 

The actual problem relies on this selector:
https://github.com/kiplaacollo/kiplaacollo.github.io/blob/b186b4345654a5a1a1a9048094bf244b0087cc43/styles.css#L176-L192

Specifically `line 178`, you're defining the width of your supporting text to be a fixed `920px`. If you remove it, the horizontal scrolling will be fixed! 🤓 🛠️